client/src/components/RegisterModal.vue

```diff
@@ -0,0 +1,266 @@
+<script setup lang="ts">
+import { ref, defineEmits } from 'vue'
+import { useDisplay } from 'vuetify'
+
+const { xs } = useDisplay()
+
+const formData = ref({
```

Comment:
Can you add a typescript type for this ref? I suggest defining this type in ``src/types/user.ts`` and import it here.